src/templates/disease-cell-line.tsx

```diff
@@ -4,132 +4,131 @@ import { Flex } from "antd";
 
```

Comment:
all I did to this page was remove a log, but the tab spacing got updated

Reply:
I also run into this a lot. We could standardize our formatting tool and settings at our next group meeting to help avoid formatting changes.